src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/DeleteInstanceV6.sql

```diff
@@ -74,11 +74,12 @@ BEGIN
     IF @@ROWCOUNT = 0
         THROW 50404, 'Instance not found', 1
         
-    -- Delete FileProperties of instance
+    -- Delete FileProperties of instance. No need to add filepath to change feed as file is deleted.
     DELETE FP
     FROM dbo.FileProperty as FP
     INNER JOIN @deletedInstances AS DI
     ON DI.InstanceKey = FP.InstanceKey
+    AND DI.Watermark = FP.Watermark
```

Comment:
Do we need the Watermark filter? Don't you want to delete all the records assosiated with a instance?